batavia/vendor/VENDORING.md

```diff
@@ -0,0 +1,69 @@
+External JavaScript Dependencies
+================================
+
+External JavaScript dependencies live in the `batavia.vendored` namespace. The
+`batavia/vendor/requirements.js` file contains a list of the currently available
+dependencies. Batavia developers consuming already vendored dependencies don't
+need to know anything else or install extra tools.
+
+The code for the vendored dependencies is bundled in the
```

Comment:
Is this the rational behind committing node_modules?

Given batavia already depends on a `make` command, can we append base tooling in there in order to get the node_modules contents?